balena/models/application.py

```diff
@@ -828,7 +828,7 @@ def is_tracking_latest_release(self, app_id):
 
         """
 
-        raw_query = "$filter=id%20eq%20'{app_id}'&$select=should_track_latest_release&$expand=should_be_running__release($select=id),owns__release($select=id&$top=1&$filter=status%20eq%20'success'&$orderby=created_at%20desc)".format(app_id=app_id)
+        raw_query = "$filter=id%20eq%20'{app_id}'&$select=should_track_latest_release&$expand=should_be_running__release($select=id),owns__release($select=id&$top=1&$filter=status%20eq%20'success'&$filter=is_final%20eq%20true&$filter=is_passing_tests%20eq%20true&$filter=is_invalidated%20eq%20true&$orderby=created_at%20desc)".format(app_id=app_id)
```

Comment:
```suggestion
        raw_query = "$filter=id%20eq%20'{app_id}'&$select=should_track_latest_release&$expand=should_be_running__release($select=id),owns__release($select=id&$top=1&$filter=status%20eq%20'success'&$filter=is_final%20eq%20true&$filter=is_passing_tests%20eq%20true&$filter=is_invalidated%20eq%20false&$orderby=created_at%20desc)".format(app_id=app_id)
```

PS: First time I'm seeing multiple `&$filter=` in the same OData subexpression!

Reply:
I changed to use `and` instead in the subexpression.

Reply:
@thgreasi, can you please check this again?

Reply:
@nghiant2710 to be clear I only requested changes for the `is_invalidated: true` -> `false`. I was find w/ the multiple $filter if as far as they worked.